### DISCUSSION TITLE
Clarion OS Logo

### DISCUSSION THREAD

**Discussion by lth7731** [bytemaster/clarion#24] [Ideas]
What does this Logo look like？ QvQ
![src=http___pic125 nipic com_file_20170320_18685668_081653494728_2 jpg refer=http___pic125 nipic](https://user-images.githubusercontent.com/35170289/110219677-b005e700-7efb-11eb-8d94-6b11ea09efc1.jpg)

**Comment by Dexaran** [comment]
There is a dedicated thread for logo discussions. Maybe you wanted to post it here: https://github.com/bytemaster/clarion/discussions/14
